.azure-pipelines/common/setup.yml

```diff
@@ -0,0 +1,17 @@
+#
+# Azure Pipelines setup configuration for react on Linux, Windows, and macOS.
+#
+
+steps:
+  # master is not fetched by default and is needed to run prettier
+  - script: git fetch origin master:master
```

Comment:
If you only need to fetch master for prettier, can you move this step next to / with the prettier step? For one, it would avoid a step showing up in every single job, especially if not all need it. It would also downplay this step, like if you moved it within your "script" step that invokes prettier.